docs/en/advanced/Command-Line-Interface.md

```diff
@@ -345,9 +565,6 @@ The `mount` command links an under storage path to an Alluxio path, and files an
 
 Options:
 
-// TODO(lu) add special things/use cases about --readonly mode?
```

Comment:
are we going to add more information about what `--readonly` does?

Reply:
I am not quite sure about what circumstances users would prefer `--readonly` so I does not add the use cases.

Reply:
@madanadit can you help out here

Reply:
`readonly` mounts are useful to prevent accidental write operations. suppose multiple Alluxio satellite clusters  mount a remote storage cluster which central source of truth, there is use case for preventing any write operation, eg. rm -R, on the satellite cluster from wiping out the remote storage.

Reply:
@madanadit Thanks! Useful documentation!

Reply:
Added, thanks!